app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- You have done well so far, but according to the project requirements, your application should ask for the confirmation of the email upon registration, i.e please include the `:confirmable` module for your devise. Please fix this 👍 👍